frontend/src/commons/Favorite.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+import { CgRemoveR } from 'react-icons/cg';
+
+export default function Favorite({ favorite, className }) {
+  return (
+    <FavoriteStyled className={className}>
```

Comment:
Maybe remove the `..Styled` postfix if there is no name clash, `FavoriteStyled` could be `FavoritesWrapper` instead. `FavoriteHeader` simply `Header` etc.